src/components/organisms/about/about.html

```diff
@@ -0,0 +1,22 @@
+<div class="about">
+  <div class="container">
+    <div class="images">
+      <div class="img d-flex">
+        <img src="../../../img/Rectangle 1.png" alt="" />
+        <img src="../../../img/Rectangle 3.png" alt="" />
```

Comment:
```suggestion
        <img src="../../../img/Rectangle1.png" alt="" />
        <img src="../../../img/Rectangle3.png" alt="" />
```
今回の課題を進める上では問題ないですが、ファイル名に空白は入れないのが望ましいです！